src/app/Instance/BridgeOverview/Components/OBDashboardTableView.tsx

```diff
@@ -83,12 +88,23 @@ export const OBDashboardTableView = (
                         </FlexItem>
                       </Flex>
                     </StackItem>
-
-                    {processor.connectors?.map((connector, rowIndex) => (
-                      <StackItem key={rowIndex} style={{ padding: "0.2rem" }}>
-                        <Label color="green">{connector}</Label>
-                      </StackItem>
-                    ))}
+                    <StackItem
+                      key={rowIndex}
+                      className={"OB-Dashboard__resource__labels"}
+                    >
+                      <LabelGroup>
+                        {processor.connectors?.map((connector, rowIndex) => (
```

Comment:
moving `StackItem` outside labels section and adding `LabelGroup` to handle space between labels.